submissions/jeffersonMormah/html/contact.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+            Contact Form Page
+        </title>
+        <meta charset="utf-8"/>
+    </head>
+    <body>
+        <h1>Ebekel Contact Form</h1>
+        <form action="#" method="POST" enctype="application/x-www-form-urlencoded">
```

Comment:
The attributes `action`, `method` and `enctype` are not needed in the form tag yet.

Reply:
@Jeffbuchi

Reply:
Hello, Its been resolved.